pkgs/tools/misc/legdur/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib
+, fetchCrate
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "legdur";
+  version = "0.3.0";
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "sha256-IKwEL5TKDDMX5AunRZI04MNAuv/w1pD39in8+j41P4Q=";
```

Comment:
```suggestion
    hash = "sha256-IKwEL5TKDDMX5AunRZI04MNAuv/w1pD39in8+j41P4Q=";
```

Reply:
What PSA am I missing here?
```
rg -A 5 "src = fetchCrate" | rg "sha256 =" | wc -l
126
```
vs
```
rg -A 5 "src = fetchCrate" | rg "hash =" | wc -l
6
```

Reply:
[This](https://github.com/NixOS/rfcs/pull/131), but it's not yet implemented, so it's only a suggestion by me and not enforceable.